korman/properties/prop_world.py

```diff
@@ -32,7 +32,8 @@ class PlasmaFni(bpy.types.PropertyGroup):
                               items=[
                                      ("linear", "Linear", "Linear Fog"),
                                      ("exp", "Exponential", "Exponential Fog"),
-                                     ("none", "None", "Use fog from the previous age")
+                                     ("exp2", "Exponential 2", "Exponential Fog 2"),
```

Comment:
I wonder if this should mention that it's distance squared?

Reply:
Is that what it does? I honestly have no clue. 😜

Reply:
@Hoikas I've updated the descriptions on all types. Thoughts?